### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by gponomarenko**
[DEMO LINK](https://gponomarenko.github.io/react_counter/)

### CODE REVIEW

**Review by artem-chornyi — APPROVED** [review]
GJ)

**Comment by gponomarenko** [comment]
> GJ)

Thanks!

File: src/components/Counter/Counter.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+  clearCounter = () => {
+    this.setState({
+      count: 0,
+    })
+  }
+
+  render() {
+    return (
+      <React.Fragment>
```
Comment (danylo-onishchuk) [inline src/components/Counter/Counter.jsx:35]: можно просто <>

File: src/App.js [hunk 2]
```diff
@@ -1,18 +1,11 @@
...
-  increase = () => {};
-
-  render() {
-    return (
-      <h1>Count: 0</h1>
-    );
-  }
-}
+const App = () => (
+  <React.Fragment>
```
Comment (danylo-onishchuk) [inline src/App.js:6]: <>

**Review by danylo-onishchuk — APPROVED** [review]

**Pull request closed without merging**